app/javascript/components/editor/components/widgets/CustomSelectWidget.tsx

```diff
@@ -149,6 +149,14 @@ class CustomSelectWidget extends React.Component<CustomSelectWidgetProps, Custom
       placeholder = `Select ${title}`
     }
 
+    if (enumOptions && enumOptions.length > 0) {
+      enumOptions.forEach((option: EnumOptionsType<RJSFSchema>) => {
+        if (option.value && option.label) {
+          selectOptions.push({ value: option.value, label: option.label })
```

Comment:
This is creating two of every option

Reply:
And then higher above, on line 127, should we change it from a check mark to "No Selection" to help avoid confusion?